### ISSUE TITLE
Idea: customizable titlebar format

### ISSUE THREAD

**Issue by jdpurcell** [jurplel/qView#538]
When I implemented #375 in my `master` branch, it got me thinking it would be nice to have a "Custom" option for the "Titlebar text" setting, which gives a textbox to specify the format, with placeholders such as the following:

```
%n - filename
%i - index in folder
%c - folder file count
%s - filesize
%w - width
%h - height
%z - zoom level
```

**Comment by jurplel** [comment]
Yeah this would be cool for sure. I think it would have to come with a settings dialog rewrite since it's already getting crowded in there,

**Comment by jdpurcell** [comment]
Implemented [here](https://github.com/jdpurcell/qView/commit/255142be8c65d8eab5a5b17ec9ba0fb8c00c54de)! Not sure if it's official-qView-worthy due to the settings UI.

**Comment by Hrxn** [comment]
Nice, thanks a lot!
